packages/osl-operator-bundle-image/generated/bundle/manifests/logic-operator-rhel8-builder-config_v1_configmap.yaml

```diff
@@ -28,4 +28,6 @@ data:
     -Djava.util.logging.manager=org.jboss.logmanager.LogManager\"\nENV JAVA_APP_JAR=\"/deployments/quarkus-run.jar\"\n"
 kind: ConfigMap
 metadata:
+  labels:
+    app.kubernetes.io/name: sonataflow-operator
```

Comment:
Shouldnt this be `logic-operator-rhel8` ?

Reply:
or not?
![image](https://github.com/user-attachments/assets/98a213ce-b291-4820-8378-9e8f23eb0b52)


Feels a bit confusing when comparing to the prod installation for 1.35 @ricardozanini 
in 1.35 the label was not there:
![image](https://github.com/user-attachments/assets/5504b668-e739-421a-bfb3-aa19978d3d74)